UserStory1/ex3test.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from ex3 import skip_sports
+
+class Testex1(unittest.TestCase):
+
+    def test_valid_(self):
+        self.assertEqual(skip_sports(2), {3: 'nnnn', 4: 'iiii'})
```

Comment:
Test outcome depends upon user inputs. 
All input required to the test should be provided as parameters. Update the `skip_sports()` function.